types/sinon/index.d.ts

```diff
@@ -775,7 +775,7 @@ declare namespace Sinon {
     }
 
     interface SinonMockStatic {
-        (): SinonExpectation;
+        (name?: string): SinonExpectation;
```

Comment:
given the other signature:

```ts
(obj: any): SinonMock;
```

i wonder if we should fix that too since `any` is actually too wide right? it can't be falsy or a string, so i guess it is `object` or something?

Reply:
lets leave it for now i think 👍 

im usually up for breaking everyone's CI to get the types right but this one seems like it could cause a lot of trouble. we can do it another time when we're feeling more chaotic 😂